platform/linuxbsd/os_linuxbsd.h

```diff
@@ -138,6 +138,8 @@ class OS_LinuxBSD : public OS_Unix {
 
 	virtual String get_system_ca_certificates() override;
 
+	virtual String get_default_android_sdk_path() const override;
```

Comment:
This feels too specific and rarely used function to add it to `OS`. Probably better to add `ifdef`s around `"export/android/android_sdk_path"` declaration instead.

Reply:
Maybe put this method to `EditorPaths`?

Reply:
Did you mean use `_WIN32`, `__linux__`, `__APPLE__` macros inside `export.cpp/register_android_exporter()` function? I don't these macros are often used in the Godot's codebase, so I thought it's not recommended to use them.

Reply:
Godot have own macros for target platofrms: `WINDOWS_ENABLED`, `MACOS_ENABLED`, `ANDROID_ENABLED`, `UNIX_ENABLED` and `WEB_ENABLED`. Note: `UNIX_ENABLED` is enabled for macOS/iOS/Android and Linux, so should be checked last.